spec/models/team_member_spec.rb

```diff
@@ -14,4 +14,29 @@
       expect(subject.name).to eq "Joe"
     end
   end
+
+  describe "#job_title" do
+    it "returns convert discipline to job tile " do
+      team_member_one = TeamMember.new(discipline: "Development")
+      team_member_two = TeamMember.new(discipline: "Delivery")
+      team_member_three = TeamMember.new(discipline: "Research")
+      team_member_four = TeamMember.new(discipline: "Service Design")
+      team_member_five = TeamMember.new(discipline: "Design")
+      team_member_six = TeamMember.new(discipline: "Operations Engineering")
+      team_member_seven = TeamMember.new(discipline: "Strategy")
+      team_member_eight = TeamMember.new(discipline: "Technical Architecture")
+      team_member_nine = TeamMember.new(discipline: "Product")
```

Comment:
These would be clearer as `developer`, `designer`, etc. It helps tracking who's who when reading the test. A suggestion, not a requirement.